src/main/java/com/purbon/kafka/topology/AccessControlManager.java

```diff
@@ -136,7 +136,7 @@ public void apply(final Topology topology, ExecutionPlan plan) {
         .forEach(
             topic -> {
               final String fullTopicName = topic.toString();
-              Set<Consumer> consumers = new HashSet<>(topic.getConsumers());
+              Set<Consumer> consumers = new LinkedHashSet<>(topic.getConsumers());
```

Comment:
I don't think this _LinkedHashSet_s are necessary here. In the end, these high-level consumer, producer, or other Service account level definitions are mapped to actions that will generate TopologyBuilderBindings (wrapper of AclBindings and RBACBindings).

To keep the "order" in the file as was your original idea, we only need to keep the order on the generated bindings, isn't?

Reply:
Well, when I wrote this code...then it was needed. Random order of consumers/producers will give random order of acls in the file (when written to the file).

But after I raised this PR you fixed the acl removal bug (#127) and the behaviour changed. So now the file is not touched if no changes. New acls are now also added to the end. However the order of acls in the file are still totally randomized after you revoke some acls (e.g. remove a consumer). 

I think we should have a predictable ordering of acls in the file. So after a small change you can easily verify manually that the .cluster-state file is correct. Will be useful in som debugging cases for example.

So instead I added one more commit to keep the ordering in the file also after acls removal. :-) 

Btw: You are right about that, right now, using HashSet here will not cause a randomized reordering (after your fix #127). But it will however still give a better ordering for many use cases; acls ordered by project -> topic -> consumer. Also future changes elsewhere might have a higher chance of reintroducing the randomization without LinkedHashSet also here.

What do you think?